Calculator App

misocernak opened the pull request.

misocernak committed:
Create kalkulator.py

misocernak committed:
Delete kalkulator.pyp

misocernak committed:
Create kalkulator.py

mrmijus commented on kalkulator.py:
```diff
@@ -0,0 +1,31 @@
+    if math_operation not in OPERATIONS:
+        return True
+    else:
+        return False
```
No need to have else. :) 
```suggestion
def check_operation(math_operation: str) -> bool:
    if math_operation not in OPERATIONS:
        return True
    return False
```

mrmijus commented on kalkulator.py:
```diff
@@ -0,0 +1,31 @@
+    #     result = num_a * num_b
+    # else:
+    #     result = num_a / num_b
+    print(result)
```
You should never push dead code (commented out code, or code that is not used anywhere) into active PR's.
You should clean this up.

mrmijus commented on kalkulator.py:
```diff
@@ -0,0 +1,31 @@
+
+    calculate(num_a, num_b, operator)
+
+main()
```
Please add this above the main function call:
`if __name__ == "__main__": `

And read about why we need to use it, and why it is useful.

mrmijus commented on kalkulator.py:
```diff
@@ -0,0 +1,31 @@
+        return False
+
+def calculate(num_a, num_b, operation):
+    result = eval(str(num_a) + operation + str(num_b))
```
I don't like this. It's a good solution, and it will solve the problem at hand, but it comes as potentially high price. This can be a very dangerous solution because it will eval **ANY code** that you pass to it, so it can be a huge security issue. It this case, it's okay, because we check the input before hand, but i wanted to you have this in mind. 

Also, you can replace eval() with **simpleeval** - a eval substitute that has less access, and therefor is  a lot safer. :) 

Read about it here: https://pypi.org/project/simpleeval/

misocernak committed:
Update kalkulator.py

Changes regarding the comments from Milos
Couldn't use simpleeval() function - it was unclear to me how to call it, should I import the module to be able to use simpleeval, tried with import ast and from simpleeval import simple_eval, nothing happens
